src/cell-templates/cell-templates.component.html

```diff
@@ -1,12 +1,12 @@
 <!-- cell templates -->
 <!-- default cell -->
 <ng-template #labelTemplate [cellTemplate]="'labelTemplate'" let-row="row" let-col="col" let-idx="rowIdx">
-  <span class="cell-content" [innerHTML]="getFieldData(row, col) || ' ' | translate | sanitize"></span>
+  <span class="cell-content" [innerHTML]="getFieldData(row, col) || 'EMPTY_CELL' | translate | sanitize"></span>

```

Comment:
The translate key should have the `ICELL_DATA_TABLE.` prefix (same below)

Reply:
valid, updating